Lib/test/test_statistics.py

```diff
@@ -1476,6 +1521,20 @@ def test_negative_error(self):
             with self.subTest(values=values):
                 self.assertRaises(exc, self.func, values)
 
+    def test_single_value_unsupported_type(self):
+        with self.assertRaisesRegex(TypeError, '^unsupported type$'):
```

Comment:
I don't see the value of this regex here: Would it be wrong if we changed the error message in the future? I prefer that we just check that a `TypeError` is raised.

Reply:
Done.